android/assets/jsons/translations/template.properties

```diff
@@ -439,11 +439,17 @@ No victory conditions were selected! =
 Mods: = 
 Extension mods = 
 Base ruleset: = 
+# Note - do not translate the colour names between «». Changing them works if you know what you're doing.
 The mod you selected is incorrectly defined! = 
+The mod you selected is «RED»incorrectly defined!«» = 
 The mod combination you selected is incorrectly defined! = 
+The mod combination you selected is «RED»incorrectly defined!«» = 
 The mod combination you selected has problems. = 
 You can play it, but don't expect everything to work! = 
+The mod combination you selected «GOLD»has problems«». = 
+You can play it, but «GOLDENROD»don't expect everything to work!«» = 
```

Comment:
Let's remove the colored tests please, this is both visually unappealing as you have said and could easily break in translation